cg/meta/workflow/rnafusion.py

```diff
@@ -149,10 +151,24 @@ def write_params_file(
             ),
         )
 
+    def get_trailblazer_config_path(self, case_id: str) -> Path:
+        """Return the path to a trailblazer config file containing Tower IDs."""
+        return Path(self.root_dir, case_id, "tower_ids.yaml")
```

Comment:
I think this should be in the trailblazer_api and called something like `get_nf_tower_config_path()`

Reply:
I like keeping it the same name for all analyses since it is then used in `add_pending_trailblazer_analysis` and it is still correct. This is the trailblazer config that is stored in the same field in the TB database regardless of it contains slurm or tower ids. As it is right now in cg it different workflows define this function in their APIs so seem reasonable to me to keep the same logic.  I would favour keeping it as it is.

Reply:
resolving and leaving as it is